integration-tests/environments/react-native-test-app/package.json

```diff
@@ -127,6 +128,12 @@
       "dependencies": [
         "../../../packages/realm/:bindgen:jsi"
       ]
+    },
+    "realm:lint-android": {
+      "command": "cd android && ./gradlew realm:lint",
+      "dependencies": [
+        "../../../packages/realm/:bindgen:jsi"
```

Comment:
Need to double-check that this dependency is actually needed as it slows down the execution quite significantly 🤔